src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -32,6 +41,15 @@
             <td>
                 {{ version.modificationDate|localizeddate('medium', 'short', app.request.locale) }}
             </td>
+            {% if isDraft is defined %}
+                <td>
+                    <a href="#" class="table-iconbutton">
```

Comment:
that won't work (I know this comes from the UI guidelines...) If you really want to fill that column, for now just write something like:

```html
<td>
    <a href="#" class="ez-button" data-disabled>Edit</a>
</td>
```